projects/plugins/jetpack/modules/sso.php

```diff
@@ -1276,6 +1279,20 @@ public function is_user_connected( $user_id ) {
 	public function get_user_data( $user_id ) {
 		return get_user_meta( $user_id, 'wpcom_user_data', true );
 	}
+
+	/**
+	 * Add styling fix for buttons within the user table.
+	 **/
+	public function jetpack_sso_user_table_button_styles() {
+		?>
+			<style>
+				.wp-list-table.users .button {
```

Comment:
Could we make this a bit more specific, to avoid making changes to other plugins that may add buttons to the interface?

Reply:
Yes good idea. I added selectors for the login as user and login as customer plugins specifically.